Projects/l0/Imu/lib/Imu/mpu_configuration.c

```diff
@@ -573,9 +573,7 @@ void mpu_setup(void)
 	     *
 	     * DMP sensor fusion works only with gyro at +-2000dps and accel +-2G
 	     */
-    hal.dmp_features = DMP_FEATURE_6X_LP_QUAT | DMP_FEATURE_TAP |
-                       DMP_FEATURE_ANDROID_ORIENT | DMP_FEATURE_SEND_RAW_ACCEL | DMP_FEATURE_SEND_CAL_GYRO |
-                       DMP_FEATURE_GYRO_CAL;
+    hal.dmp_features = DMP_FEATURE_6X_LP_QUAT | DMP_FEATURE_TAP | DMP_FEATURE_ANDROID_ORIENT | DMP_FEATURE_SEND_RAW_ACCEL | DMP_FEATURE_SEND_CAL_GYRO | DMP_FEATURE_GYRO_CAL;
```

Comment:
In this case, do we need a logical OR (operator | )  or a or (operator || ) ?

Perhaps we should put OR operators at line beginning.  So, after clang format processing, the conditions should be on different lines.